### DISCUSSION TITLE
Pocket option API

### DISCUSSION THREAD

**Discussion by WayneFinance** [community/community#136253] [Programming Help]
Has anyone created or creating an AI for pocket option?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @WayneFinance , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
